src/App.jsx

```diff
@@ -95,6 +109,7 @@ const App = () => {
       <CssBaseline />
       <AuthContext.Provider value={authContext}>
         <ThemeProvider theme={theme}>
+          {backendError && <ErrorPanel fixed error={backendError} />}
```

Comment:
I think we can implement the logic in `Main.jsx` so that we are able to use the theme object from material UI. Also, in order to only show the backend error, we could do something like that in `Main.jsx`:

```jsx
// ...

backendError: {
    marginTop: // add some margin top
},

// ...

return backendError ? (
 // <ErrorPanel fixed className={classes.backendError} error={...} />
) : (
 // 
);
```

Reply:
I ended up going a bit further here and disabling most of the UI when the backend is unavailable.